.travis/script.sh

```diff
@@ -26,7 +26,7 @@ if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
     macdeployqt $BUNDLE_NAME
 
     # Fix other deps that macdeployqt missed
-    wget -c -nv https://github.com/arl/macdeployqtfix/raw/master/macdeployqtfix.py
+    curl -sLOC - https://github.com/arl/macdeployqtfix/raw/master/macdeployqtfix.py
```

Comment:
> Use "-C -" to tell curl to automatically find out where/how to resume the transfer. It then uses the given output/input files to figure that out.

What purpose does this option serve here?

Reply:
It's just replicating the existing behaviour of `wget -c -nv`.

Reply:
Hm, I wonder if wget retries a failed transfer. In that case it would make a lot of sense. But does curl can do that too?

Reply:
> Note that you don't need to specify this option if you just want the current invocation of Wget to retry downloading a file should the connection be lost midway through. This is the default behavior.

So wget does retry, 20 times by default. Curl does not:

> If a transient error is returned when curl tries to perform a transfer, it will retry this number of times before giving up. Setting the number to 0 makes curl do no retries (which is the default)

There are also some remarks in the wget man pages about `-c` in v1.7+ which can lead to a download failure. Thus I think it should not be used and I'd drop the `-C -` for curl. We might want to add `--retry 3` or something though.

Reply:
It's more so curl won't download the file repeatedly when performing subsequent builds.

Reply:
Actually, this does not replicate wget's behavior... In case of an HTTP error, curl will just save the error reply to a file and return 0 as exit code (success). wget fails in such a case. I think we should thus use the following replacement:

```suggestion
    curl -fLOSs --retry 3 https://github.com/arl/macdeployqtfix/raw/master/macdeployqtfix.py
```

- `-f` fail on error
- `-L` follow redirects
- `-O` output to file
- `-S` print errors
- `-s` but be silent otherwise

Reply:
You forgot the `-C` flag to avoid redownloading over and over btw.

Reply:
I actually removed it on purpose, because it doesn't do anything here. On each CI run, a clean environment is used, hence all files which are not part of the build image need to be downloaded.